corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -458,11 +458,17 @@ hqDefine("cloudcare/js/form_entry/form_ui", function () {
             return !self.isSubmitting() && self.erroredQuestions().length === 0;
         });
 
+        self.getSubmitTranslation = function () {
+            var translations = self.translations;
+            const [key, _] = Object.entries(translations).find(([k]) => k.includes("submit_label"));
+            return ko.toJS(translations[key]);
+        };
+
```

Comment:
this goes with this formplayer pr: https://github.com/dimagi/formplayer/pull/1512